Gordon360/Models/CCT/Housing/Resident_Advisor.cs

```diff
@@ -0,0 +1,66 @@
+﻿// <auto-generated> This file has been auto generated by EF Core Power Tools. </auto-generated>
+#nullable disable
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using Microsoft.EntityFrameworkCore;
+
+namespace Gordon360.Models.CCT;
+
+[Table("Resident_Advisor", Schema = "Housing")]
+public partial class Resident_Advisor
+{
+    [Key]
+    public int Ra_ID { get; set; }
+
+    [Required]
+    [StringLength(100)]
+    [Unicode(false)]
+    public string Fname { get; set; }
+
+    [Required]
+    [StringLength(100)]
+    [Unicode(false)]
+    public string Lname { get; set; }
+
+    public int? Hall_ID { get; set; }
+
+    public int? Floor { get; set; }
+
+    public int? Room_Num { get; set; }
+
```

Comment:
This data is all in J1, so you shouldn't try to maintain it separately. If you add a `Gordon_ID` column containing the RA's Jenzabar Id, you can trivially create a view that combines the data from this table with the data from J1 as needed.

Reply:
Hall ID is possibly a special case. We do store hall information in J1, in the Building_Master table. However, halls are not stored separately from other buildings, so you'd need to create a view of the J1 buildings that are halls. That still might be a better solution in the long term though. That way, you can use the RA's building code in their J1 data to link them to their Hall.

Reply:
If you want help accessing J1 data or figuring out how to utilize it well in your project, let me know! I'd be happy to work with y'all on that.

Reply:
> This data is all in J1, so you shouldn't try to maintain it separately. If you add a `Gordon_ID` column containing the RA's Jenzabar Id, you can trivially create a view that combines the data from this table with the data from J1 as needed.

Is the `student` view in the MyGordon database where this data would be pulled from to make a local view for the context of the Housing page?